templates/papertrail/papertrail-logspout-daemonset.yml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: logspout-papertrail-secrets
+  namespace: '{{ k8s_papertrail_logspout_namespace }}'
```

Comment:
if we allow customization of the namespace, do we have to add a block to create that namespace?

Reply:
Good point. I added a comment in defaults.yml to note this.